README.md

```diff
@@ -1,91 +1,205 @@
-Successfully created secret `db-passwords` in namespace `default`.
-Successfully created secret `auth-secrets` in namespace `default`.
-All required secrets have been successfully created!
-```
-
-**Optional:** If you're going to be triggering connector jobs from the admin-api (used by the admin UI), create the role binding this needs:
+cd terraform/magda
+```
 
 ```bash
-kubectl apply -f role-binding.yaml
+gcloud iam service-accounts keys create key.json --iam-account=$SERVICE_ACCOUNT_EMAIL
 ```
 
-6. Add the magda chart repo to helm
+You will now have a `key.json` file in `terraform/magda`, containing a private key. We suggest you put this somewhere safe like a password manager. DO NOT CHECK IT INTO SOURCE CONTROL.
+
+#### 10. Grant service account permission:
+
+Grant `editor` role to your service account:
+
 ```bash
-helm repo add magda-io https://charts.magda.io
+gcloud projects add-iam-policy-binding $PROJECT_ID --member serviceAccount:$SERVICE_ACCOUNT_EMAIL --role roles/editor
 ```
 
-7. Install magda
+Grant `k8s admin` role to your service account:
 ```bash
-helm upgrade magda magda-io/magda --wait --timeout 30000 --install -f config.yaml
+gcloud projects add-iam-policy-binding $PROJECT_ID --member serviceAccount:$SERVICE_ACCOUNT_EMAIL --role roles/container.admin
 ```
 
-This will take a while for it to get everything set up. If you want to watch progress, run `kubectl get pods -w` in another terminal.
+#### 11. Clone this repo & Initiate Terraform
+ and run:
 
-8. Once helm is finished, run `kubectl get services -w` and wait for `gateway` to receive an external IP. It'll look something like this on kubectl or docker for desktop. **Note that minikube doesn't support this**, you'll just have to run `minikube ip` and keep refreshing it with the node port below until it comes up.
+```bash
+terraform init
+```
+
+After a bit of waiting you should get this message:
 
 ```
-gateway                           LoadBalancer   10.102.57.74     123.456.789.123     80:31519/TCP        1m
+Terraform has been successfully initialized!
+
+You may now begin working with Terraform. Try running "terraform plan" to see
+any changes that are required for your infrastructure. All Terraform commands
+should now work.
+
+If you ever set or change modules or backend configuration for Terraform,
+rerun this command to reinitialize your working directory. If you forget, other
+commands will detect it and remind you to do so if necessary.
 ```
 
-At this point you should be able to go to `http://<external ip>` in your browser and see the Magda UI. Note that the search won't work until it's finished indexing regions - to see the progress of this, run `kubectl logs -f -lservice=indexer`. Unless you've got a lot of processing power this will take quite a while - sorry! We're working on making it better.
+#### 12. Edit terraform config
+
+Edit [terraform/magda/terraform.tfvars](./terraform/magda/terraform.tfvars) and supply the follow parameters:
```

Comment:
This shouldn't be a link to a file. It should just be in backticks.
```suggestion
Edit `terraform/magda/terraform.tfvars` and supply the follow parameters:
```

Reply:
Does it give you any troubles to find the file?
It supposes to provide a reference point back to the repository.